SimG4CMS/Calo/interface/CaloHitID.h

```diff
@@ -39,7 +39,7 @@ class CaloHitID {
   int          theTrackID;
   int          theTimeSliceID;
   uint16_t     theDepth;
-  int          timeSliceUnit;
+  double       timeSliceUnit;
```

Comment:
It is better be a double (in any case this is a transient object)
________________________________
From: David Lange [notifications@github.com]
Sent: 04 October 2015 19:27
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcx31 Add simulation code for HGCal and single layer fast timing device (#11498)


In SimG4CMS/Calo/interface/CaloHitID.h<https://github.com/cms-sw/cmssw/pull/11498#discussion_r41099973>:

> @@ -39,7 +39,7 @@ class CaloHitID {
>    int          theTrackID;
>    int          theTimeSliceID;
>    uint16_t     theDepth;
> -  int          timeSliceUnit;
> +  double       timeSliceUnit;


@bsunanda<https://github.com/bsunanda> - could this be a float?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/11498/files#r41099973>.